lesson10/main.js

```diff
@@ -52,12 +52,11 @@ window.onload = () => {
   const callLists = async () => {
     try {
       const listsContents = await fetchedData;
-      loaded();
       createNewList(listsContents);
     } catch (error) {
       console.error(error);
     } finally {
```

Comment:
Thank you for fixing this. I've already "Approve" it once, and I'll leave it to you to fix it or not, but this process alone will not display anything on the screen when an error occurs.
Here is the code I imagine.
 [Please take a look if you like.](https://codesandbox.io/s/lesson10-forked-8nbq8?file=/main.js).

Since the purpose of this assignment was to write "at the end", I wrote the exact same process as yuka-san and got approved during the teacher's review, so please take the sandbox as an idea😊

Reply:
@443n0511 
Thank you for your advice! I thought that idea was nice, it's like a user friendly! I would like to adopt that.
My code became nice with your advice, much appreciated! :) :)